split.Console/Interfaces/IPlayerInterface.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by an 
+    /// object that will encapsulate all player-related behavior. 
+    /// </summary>
+    public interface IPlayerInterface : ICardInterface
```

Comment:
Agreed. This is done to be able to get a list of ```ICardInterface``` objects. ```IHaveHand``` would work also, I think. It would just be another layer of abstraction because ```IHaveHand``` would extend ```ICardInterface```.